crowbar_framework/app/models/ceilometer_service.rb

```diff
@@ -127,6 +137,10 @@ def validate_proposal_after_save proposal
 
     validate_minimum_three_nodes_in_cluster(proposal)
 
+    unless proposal["deployment"]["ceilometer"]["elements"]["ceilometer-hyperv-agent".empty?] || hyperv_available?
+      validation_error("Hyper-V support is not available.")
```

Comment:
We could be more verbose here, but I guess the error message gives enough information and I'd like to be consistent with the nova-compute-hyperv check.